config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json

```diff
@@ -29,23 +16,26 @@
     "description": "Ridge CSI image",
```

Comment:
Can you add a better description for this? Something along the lines of "Location/URL of the Ridge CSI image registry".